src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -154,6 +154,10 @@
 
 "NotificationSettings_NavTitle" = "Notifications";
 
+"NotificationSettings_onImageDescription" = "Eine Frau erhält eine Mitteilung von ihrer Corona-Warn-App.";
+
+"NotificationSettings_offImageDescription" = "Eine Frau kann keine Mitteilung von ihrer Corona-Warn-App erhalten.";
+
```

Comment:
Translation files are mixed up. German text in English file, and English text in German file.

Reply:
This is fixed now